pkg/vault/test/foundry/utils/BasePoolTest.sol

```diff
@@ -70,30 +70,29 @@ abstract contract BasePoolTest is BaseVaultTest {
         (, , uint256[] memory balances, ) = vault.getPoolTokenInfo(address(pool));
 
         for (uint256 i = 0; i < poolTokens.length; ++i) {
-            // Tokens are transferred from lp
+            // Tokens are transferred from lp.
             assertEq(
                 defaultBalance - poolTokens[i].balanceOf(lp),
                 tokenAmounts[i],
                 string.concat("LP: Wrong balance for ", Strings.toString(i))
             );
 
-            // Tokens are stored in the Vault
+            // Tokens are stored in the Vault.
             assertEq(
                 poolTokens[i].balanceOf(address(vault)),
                 tokenAmounts[i],
                 string.concat("LP: Vault balance for ", Strings.toString(i))
             );
 
-            // Tokens are deposited to the pool
+            // Tokens are deposited to the pool.
             assertEq(
                 balances[i],
                 tokenAmounts[i],
                 string.concat("Pool: Wrong token balance for ", Strings.toString(i))
             );
         }
 
-        // should mint correct amount of BPT poolTokens
-        // Account for the precision loss
+        // Should mint correct amount of BPT poolTokens, accounting for the precision loss.
```

Comment:
```suggestion
        // Should mint correct amount of BPT poolTokens. DELTA accounts for precision loss.
```

Not sure if the english is correct, but looks like there are 2 information in this sentence. 
* AssertApproxEqAbs checks if the correct amount of BPTs was issued.
* The precision loss was not accounted by the function that emitted the BPTs, but by the check, using the DELTA variable.

Reply:
Corrected that in a couple places (including one where it should have been "burn")